cypress/support/commands.js

```diff
@@ -29,13 +29,39 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 addMatchImageSnapshotCommand();
 
 Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+  cy.get(`[data-cy^="${selector}"]`);
 });
 
+// eslint-disable-next-line max-len
 Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/users', {
-    email,
-    username,
-    password
+  cy.request('POST', '/api/users', {
+    user: {
+      email,
+      username,
+      password
+    }
+  });
+});
+
+// eslint-disable-next-line max-len
+Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+  cy.request('POST', '/api/users', {
```

Comment:
The `login` command seems to be using the same API endpoint as `register`. Ensure that the `login` command is correctly implemented to authenticate the user, possibly using a different endpoint for login.